src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -208,14 +220,21 @@ protected void initialize() {
 
             @Override
             protected void execute() {
-                SmartDashboard.putNumber("Drive Angle", angleCorrection);
+                SmartDashboard.putNumber("Drive Angle Error", drivePidController.getError());
                 m_drive.arcadeDrive(0.0, angleCorrection);
 
             }
 
             @Override
             protected boolean isFinished() {
-                return drivePidController.onTarget();
+                if (drivePidController.onTarget()) {
+                    iOnTarget++;
+                }
+                if (iOnTarget >= 10) {
```

Comment:
Let's make this a preference, or at least some kind of variable with a name that explains what it's for